__tests__/app.test.js

```diff
@@ -54,5 +54,158 @@ describe('app routes', () => {
+        {
+          'name': 'Prova California Table',
+          'image_url': 'https://s3-media3.fl.yelpcdn.com/bphoto/E1OYsOzuhh8FZ63k1_Jtvw/o.jpg',
+          'price': '$$',
+          'rating': 4,
+          'url': 'https://www.yelp.com/biz/prova-california-table-morgan-hill?adjust_creative=9N4qQ7DERIKHI9lFQiFDIA&utm_campaign=yelp_api_v3&utm_medium=api_v3_business_search&utm_source=9N4qQ7DERIKHI9lFQiFDIA'
+        },
+        {
+          'name': 'CreAsian',
+          'image_url': 'https://s3-media1.fl.yelpcdn.com/bphoto/F60ABs41LLChBDEGu4jFKQ/o.jpg',
+          'price': '$$',
+          'rating': 4,
+          'url': 'https://www.yelp.com/biz/creasian-morgan-hill?adjust_creative=9N4qQ7DERIKHI9lFQiFDIA&utm_campaign=yelp_api_v3&utm_medium=api_v3_business_search&utm_source=9N4qQ7DERIKHI9lFQiFDIA'
+        },
+        {
+          'name': 'La Nina Perdida',
+          'image_url': 'https://s3-media3.fl.yelpcdn.com/bphoto/M07R8l7QTtmbhs92Ci4rbQ/o.jpg',
+          'price': '$$',
+          'rating': 4,
+          'url': 'https://www.yelp.com/biz/la-nina-perdida-morgan-hill?adjust_creative=9N4qQ7DERIKHI9lFQiFDIA&utm_campaign=yelp_api_v3&utm_medium=api_v3_business_search&utm_source=9N4qQ7DERIKHI9lFQiFDIA'
+        },
+        {
+          'name': 'Sicilia In Bocca',
+          'image_url': 'https://s3-media1.fl.yelpcdn.com/bphoto/ef7YY6Soc9ZLB_Mcw_u3Rw/o.jpg',
+          'price': '$$',
+          'rating': 4,
+          'url': 'https://www.yelp.com/biz/sicilia-in-bocca-morgan-hill-2?adjust_creative=9N4qQ7DERIKHI9lFQiFDIA&utm_campaign=yelp_api_v3&utm_medium=api_v3_business_search&utm_source=9N4qQ7DERIKHI9lFQiFDIA'
+        },
+        {
+          'name': 'Yolked Extreme Breakfast',
+          'image_url': 'https://s3-media4.fl.yelpcdn.com/bphoto/YVI7HlG6pjlqTVkCiqVqaQ/o.jpg',
+          'price': '$$',
+          'rating': 4.5,
+          'url': 'https://www.yelp.com/biz/yolked-extreme-breakfast-morgan-hill?adjust_creative=9N4qQ7DERIKHI9lFQiFDIA&utm_campaign=yelp_api_v3&utm_medium=api_v3_business_search&utm_source=9N4qQ7DERIKHI9lFQiFDIA'
+        },
+        {
+          'name': 'Big Grill BBQ & Teriyaki',
+          'image_url': 'https://s3-media4.fl.yelpcdn.com/bphoto/5D2_Y9em9MidAZWAcoPy6Q/o.jpg',
+          'price': '$$',
+          'rating': 4.5,
+          'url': 'https://www.yelp.com/biz/big-grill-bbq-and-teriyaki-morgan-hill?adjust_creative=9N4qQ7DERIKHI9lFQiFDIA&utm_campaign=yelp_api_v3&utm_medium=api_v3_business_search&utm_source=9N4qQ7DERIKHI9lFQiFDIA'
+        },
+        {
+          'name': 'I Do Wedding Cakes',
+          'image_url': 'https://s3-media4.fl.yelpcdn.com/bphoto/lKX9TpYlPEi0hyAZhNR6FQ/o.jpg',
+          'price': '$$',
+          'rating': 5,
+          'url': 'https://www.yelp.com/biz/i-do-wedding-cakes-morgan-hill-3?adjust_creative=9N4qQ7DERIKHI9lFQiFDIA&utm_campaign=yelp_api_v3&utm_medium=api_v3_business_search&utm_source=9N4qQ7DERIKHI9lFQiFDIA'
+        }
+      ];
+
+      const data = await fakeRequest(app)
```

Comment:
I like that you tested the route, but for this lab, we were looking for you to test your munging functions.  You did this for the location munge function that you made perfectly, but for points on weather and yelp, you should test your functions instead.